ena-linkage/main.py

```diff
@@ -171,14 +151,17 @@ def run_api_call(specimen_data: Dict) -> List[Dict[str, str]]:
         response = requests.get(sample_query)
         response_json = json.loads(response.content)
         if len(response_json) > 0:
-            check_result(response_json, result_list, sequence_query, specimen_data)
+            check_result(response_json, result_list, sequence_query,
+                         specimen_data)
             return result_list
         else:
-            logging.info(f'No relevant identifiers found for specimen: {specimen_data["ods:id"]}')
+            logging.info(
+                f'No relevant identifiers found for specimen: {specimen_data["ods:id"]}')
```

Comment:
I think it should be specimen_data["ods:ID"] (capitals)

Reply:
good catch :mag: